packages/wallets/src/okx/okx.page.ts

```diff
@@ -73,7 +73,9 @@ export class OkxPage implements WalletPage {
         await this.onboardingPage.importWalletButton.waitFor({
           timeout: 3000,
         });
-        await this.onboardingPage.firstTimeSetup();
+        await this.onboardingPage.firstTimeSetup(
+          this.extensionUrl + this.config.COMMON.EXTENSION_START_PATH,
```

Comment:
wdyt 
- to add class var like `extensionHomeUrl`( or like this) and use it here and in `goto()`?
- pass `extensionHomeUrl` for `firstTimeSetup() + ` - `await this.onboardingPage.firstTimeSetup(extensionHomeUrl + ''#/import-with-seed-phrase-and-private-key'' )` and then in in `firstTimeSetup` `use await this.page.goto()`